src/Form/Select.svelte

```diff
@@ -54,24 +84,49 @@
     bottom: 0 !important;
```

Comment:
How come so many `!important` rules?

Reply:
They came with the select - I just left them alone! Don't anger the CSS gods!